code/modules/gear_presets/clf.dm

```diff
@@ -165,29 +174,42 @@
 
 /datum/equipment_preset/clf/specialist/load_gear(mob/living/carbon/human/H)
 
-	//specific armor for specialist
-	H.equip_to_slot_or_del(new /obj/item/clothing/under/colonist/clf(H), WEAR_BODY)
+	//jumpsuit and their webbing
+	var/obj/item/clothing/under/colonist/clf/CLF = new()
+	var/obj/item/clothing/accessory/storage/webbing/W = new()
+	CLF.attach_accessory(H, W)
+	W.hold.storage_slots = 5
```

Comment:
make a special variant rather than adjusting the slots

Reply:
done, also cleaned up some UPP stuff that did the same thing